season/the_basics.md

```diff
@@ -0,0 +1,170 @@
+# The Basics
+
+Answer the questions below. Feel free to use this [markdown cheat sheet](https://guides.github.com/pdfs/markdown-cheatsheet-online.pdf) to help with formatting!
+
+## Question 1
+
+```ruby
+a = 5
+b = 4
+a = b
+```
+
+At the end of this program, what are the values of a and b? Try to explain, in plain English, how you got your answer for each one.
+
+**Answer 1:**
+a = 4 and b = 4 because with the last line of command we set the variable a equal to b which is 4.
+
+## Question 2
+
+```ruby
+e = 3
+f = 9
+g = e + f
+f == g
+e = 3
+```
+
+At the end of this program, what are the values of e, f, and g? Try to explain, in plain English, how you got your answer for each one.
+
+**Answer 2:**
+e = 3, f = 9, g = 12 because 9 + 3 is 12. Therefore, g = 12. "f == g" checks if the statement is true but does not change the value.
+
+
+## Question 3
+
+```ruby
+weather = "cloudy"
+weather == "cloudy"
+```
+
+What is the difference between these two statements? Explain your answer.
+
+**Answer 3:**
+"weather = "cloudy"" sets the weather as cloudy whereas "weather == "cloudy"" only checks if it is true.
+
```

Comment:
Checks if what is true?

Reply:
Also, if you want to make code blocks in your answer, wrap your code in the little ticks (``) in the top left corner:
`weather == "cloudy"`